netdeploy/remote/deployedNetwork.go

```diff
@@ -486,9 +500,14 @@ func getGenesisAlloc(name string, allocation []bookkeeping.GenesisAllocation) bo
 	return bookkeeping.GenesisAllocation{}
 }
 
-func keypair() *crypto.SignatureSecrets {
+// keypair returns a random key, unless deterministic is true, which will set i as the seed for key generation.
+func keypair(i uint64, deterministic bool) *crypto.SignatureSecrets {
```

Comment:
```suggestion
type keypairSeed uint64
const invalidSeed keypairSeed = math.MaxUint64
// keypair returns a random key, unless deterministic is true, which will set i as the seed for key generation.
func keypair(s keypairSeed) *crypto.SignatureSecrets {
```

Reply:
a special value to use crypto.random seed and not the provided value. Although it could be better to have two functions `keypairDeterm` and `keypairRandom`